third_party/2and3/werkzeug/exceptions.pyi

```diff
@@ -1,152 +1,159 @@
-from typing import Any, Dict, NoReturn
+from typing import Any, Dict, NoReturn, Optional, Protocol, Type, Union
+
+from wsgiref.types import WSGIEnvironment
+from werkzeug.wrappers import Response
+
+class _EnvironContainer(Protocol):
```

Comment:
Yeah, it sounds a bit better. I might have done a batter job and maybe think the name through. Just thought that since you actually suggested that than it's a name you will be happy about ;-)